src/components/GlobalLearningResourcesPage/GlobalLearningResourcesFilters.tsx

```diff
@@ -35,7 +34,10 @@ const GlobalLearningResourcesFilters: React.FC<
     _event: React.FormEvent<HTMLInputElement>,
     value: string
   ) => {
-    setInputValue(value);
+    setLoaderOptions({
```

Comment:
you should use the callback to prevent some out-of-sync issues:
```jsx
setLoaderOptions(prev => ({...prev, 'display-name': value})
```

Reply:
There is no display name that would match multiple empty spaces, therefore no quickstarts are displayed.